product-scenarios/backend-service/ballerinaService/resources/json/request/tmp.json

```diff
@@ -0,0 +1 @@
}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"}},{"request":{"symbol":"WSO2"
```

Comment:
Let's remove this temporary json message

Reply:
done